### DISCUSSION TITLE
Why does the Game data class have not application id

### DISCUSSION THREAD

**Discussion by Idontknow7777** [Rapptz/discord.py#6156] [Ideas]
Why does the Game data class have not application id?

**Comment by Nemika-Haj** [comment]
Please read this, there is no application ID for the Game class.
https://discordpy.readthedocs.io/en/latest/api.html?highlight=game#discord.Game

**Reply by Idontknow7777** [reply]
Yeah I know but it would be better if it did

**Reply by Nemika-Haj** [reply]
What would you get the ID of? Also, why is this in the Q&A category? There is a bright **Ideas** section.

**Reply by Idontknow7777** [reply]
it just helps identify the app since people can change the names

**Reply by Nemika-Haj** [reply]
Well, then make a thread in the **Ideas** section and close this one.

**Comment by Rapptz** [comment]
`Game` is a stripped down version for memory optimisation purposes. I don't believe most games come with an application_id so that'd fall under `Activity`.
